lighthouse-core/test/runner-test.js

```diff
@@ -38,6 +38,11 @@ describe('Runner', () => {
     };
   };
 
+  const runGatherAndAudit = async (gatherFn, opts) => {
```

Comment:
it's definitely a bit weird to refactor the runner interface but then not have any changes in its unit tests :) These are kind of `lighthouse-core/index.js` tests now instead.

- what are the new use cases being opened up by exposing both `gather` and `audit` in Runner? #13364 basically
- are those cases covered by tests?

Reply:
Ya we could have a test that does multiple calls to `gather` followed by multiple corresponding calls to `audit`